gr-utils/modtool/cli/help.py

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright 2018 Free Software Foundation, Inc.
+#
+# This file is part of GNU Radio
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+#
+""" Returns information about a module """
+
+
+import subprocess
+import click
+
+@click.command(name="help")
+def cli():
+    """
+    Display help for gr_modtool (equivalent to `gr_modtool --help`).
+    """
+    try:
+        result = subprocess.run(
+            ["gr_modtool", "--help"],
```

Comment:
that only works if `gr_modtool` is in the `$PATH` (`%path%` on windows); that's often not the case in practice!

Would it be possible to use `click.get_current_context().get_help()`? or `get_current_context().parent.get_help()`?